table-api/src/main/java/io/deephaven/api/agg/spec/AggSpecUnique.java

```diff
@@ -56,25 +64,32 @@ public final String description() {
      *
      * @return Whether to include nulls
      */
-    @Default
-    public boolean includeNulls() {
-        return false;
-    }
+    @Nullable
+    public abstract Boolean includeNulls();
 
     /**
      * The output value to use for groups that don't have a single unique input value.
      *
      * @return The non-unique sentinel value
      */
-    @Nullable
-    @Default
-    public Object nonUniqueSentinel() {
-        return null;
+    public abstract Optional<Object> nonUniqueSentinel();
+
+    public final boolean includeNullsOrDefault() {
```

Comment:
@niloc132 We should discuss this as a group. Do we really want clients to accept server side defaults in scenarios where we have a wire-compatible APIs that mean something else on the server? I really don't think that's best.